dev/com.ibm.ws.security.wim.adapter.ldap/src/com/ibm/ws/security/wim/adapter/ldap/LdapConnection.java

```diff
@@ -370,7 +425,7 @@ public void initialize(Map<String, Object> configProps) throws WIMException {
      */
     private void initializeContextManager(Map<String, Object> configProps) throws WIMException {
 
-        iContextManager = new ContextManager();
+        iContextManager = new ContextManager(iReposId, kerberosService);
```

Comment:
ContextManager will need to take in the new KerberosService as well, or you will need to new up a new one.